code/script.js

```diff
@@ -6,4 +6,8 @@ function toggle() {
 }
 
 // Selects an HTML element, and calls a function which will be executed when the element is clicked.
+
 document.getElementById("section1").onclick = toggle
+document.getElementById("section2").onclick = toggle
+document.getElementById("section3").onclick = toggle
+document.getElementById("section4").onclick = toggle
```

Comment:
Nice and clean 👍 You could also (if you want), break these out into variables, like this:
```
const section1 = document.getElementById("section-1")
section1.onclick = toggle
```